bmh_admin_portal_backend/lambdas/workspaces_api_resource/workspaces_api_resource_handler.py

```diff
@@ -264,10 +273,17 @@ def _workspaces_post(body, email):
 
     # Generate email address for root account
     email_domain = os.environ.get("email_domain", None)
-    if email_domain is None:
+    if email_domain:
```

Comment:
Hi Urvi, just wondering why the "is None" is removed here? Looks like the code will throw the error if email_domain is successfully populated?

Reply:
I think the right condition you're looking for is 

```suggestion
    if not email_domain:
```